src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_ReportRepairs.php

```diff
@@ -16,6 +16,29 @@ function __construct()
     }
 
     /**
+     * Prefixes a given report with Broken: to let users know problematic reports
+     * @param $id
+     */
+    public function markReportBroken($id)
```

Comment:
It'd make more sense to keep them in their respective class and set them as static, which was why I made this change at this point. Once they are static however, we would need to rework the Logging portion of this however, which would be useful anyway. 

I'm reverting them for now.